resources/views/hr/job/edit.blade.php

```diff
@@ -55,7 +55,15 @@
                             @foreach (config('hr.opportunities.domains') as $domain => $label)
                                 <option value="{{ $domain }}" {{ old('domain', $job->domain) == $domain ? 'selected' : '' }}>{{ $label }}</option>
                             @endforeach
-                        </select>
+                        </select>   
+                    </div>
+                    <div class="col-md-3 form-group">
+                        <label for="start_date" class="fz-14 leading-none text-secondary mb-1">Start Date></label>
+                        <input type="date" class="form-control" id="start_date" name="start_date" value="{{ old('start_date' , $job->start_date)}}">     
```

Comment:
Please fix formatting
```suggestion
                        <input type="date" class="form-control" id="start_date" name="start_date" value="{{ old('start_date' , $job->start_date) }}">     
```